frontend/public/style/_common.scss

```diff
@@ -107,7 +107,7 @@
 .co-section-title {
   font-size: 20px;
   font-weight: 600;
-  margin: 0 0 30px 0;
+  margin: 0 0 20px 0;
```

Comment:
This is going to change the margin on all the pages, even those without explanations. I'm not sure we want that.

Reply:
@jennyhaines Do you want the spacing changes to affect all the headers (even when there is no description)? This reduces the spacing between a header and its content.

Reply:
Oh, I see. No, I don't think that the margin should be changed in instances when there is no description. Is there a way to add an additional spacer between the description and the header without applying directly to the header or the description?

Reply:
@jennyhaines I can add extra spacing between the description and header only when the description is there. I'll change this to reduce spacing between the header and content when the description is visible.

Reply:
@jennyhaines I talked to Joe. If we reduce the space between “Container hello-openshift” and the form fields conditionally when the description is there, it'll make the site inconsistent. Do you want it changed globally instead?

I added the space between the instructional text and “Container hello-openshift" so it's only there when the instructional text is there.